stackslib/src/config/chain_data.rs

```diff
@@ -808,6 +809,87 @@ EOF
         );
     }
 
+    #[test]
+    #[cfg(windows)]
+    fn test_get_unconfirmed_commits() {
+        let shell_code = r#"@echo off
+(
+echo [
+echo  {
+echo   "txid": "73c318be8cd272a73200b9630089d77a44342d84b2c0d81c937da714152cf402",
+echo   "burn": 555000,
+echo   "address": "1FCcoFSKWvNyhjazNvVdLLw8mGkGdcRMux",
+echo   "input_txid": "ef0dbf0fc4755de5e94843a4da7c1d943571299afb15f32b76bac5d18d8668ce",
+echo   "input_index": 3,
+echo   "pox_addrs": [
+echo       "0014db14133a9dbb1d0e16b60513453e48b6ff2847a9",
+echo       "a91418c42080a1e87fd02dd3fca94c4513f9ecfe741487"
+echo   ]
+echo  }
+echo ]
+)
+"#;
+        let path = "/tmp/test-get-unconfirmed-commits.bat";
```

Comment:
This works in Git Bash but might not work in PowerShell, perhaps a more portable way could be
```suggestion
        let path = std::env::temp_dir().join("test-get-unconfirmed-commits.bat");
```